tweets/views.py

```diff
@@ -1,5 +1,6 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
 from django.views.generic import TemplateView
 
 
-class HomeView(TemplateView):
+class HomeView(TemplateView, LoginRequiredMixin):

```

Comment:
```suggestion
class HomeView(LoginRequiredMixin, TemplateView):
```
Mxin は View より左に書きましょう！左側に書いた基底クラスの方が優先されます．（メソッド解決順序という）